### ISSUE TITLE
Not working on xubuntu on rpi3

### ISSUE THREAD

**Issue by tanjid440** [Meticulus/tomxplayer#3]
please make this thing work with **xubuntu**...

**Comment by Meticulus** [comment]
tomxplayer, is build specifically for Raspbian Stretch and Ubuntu MATE 16.04 LTS. I make no claims as to whether it will work other desktop environments.

However, of the two, the raspi package is less dependent on the desktop environment. 

Did you try that package?

What exactly is the problem?

**more info needed...**

**Comment by tanjid440** [comment]
Dear Sir,

Yeah, I've tried that too. But no result. Later I discovered there was _something wrong with the omxplayer itself_ . Since, the omxplayer is faulty tomxplayer has nothing to do. If omxplayer works, tomxplayer will definitely work (I believe). But i gave up on **xubuntu**. Rather **Arch Linux** is a very good distro for raspberry pi 3 platform! So, I am requesting you with all due respect  to port **tomxplayer on arch linux** for raspberry pi 3.

Thank You.
